nni/algorithms/compression/v2/pytorch/pruning/basic_pruner.py

```diff
@@ -440,12 +441,12 @@ def reset_tools(self):
         else:
```

Comment:
the function name `reset_tools` is not very good. what is this function used for?

Reply:
yes, try to remove it, it is used for resetting the `DataCollector`, `MetricCalculator`, `SparsityCalculator` when resetting the pruner.